sktime/datatypes/_utilities.py

```diff
@@ -210,7 +262,7 @@ def update_data(X, X_new=None):
         return X_new.combine_first(X)
 
 
-GET_LATEST_WINDOW_SUPPORTED_MTYPES = [
+GET_WINDOW_SUPPORTED_MTYPES = [
```

Comment:
```suggestion
WINDOW_SUPPORTED_MTYPES = [
```

Reply:
same - refers to the function `get_window`

Reply:
ok